swagger/v1/move.json

```diff
@@ -35,7 +35,7 @@
           },
           "status": {
             "type": "string",
-            "enum": ["requested", "cancelled", "completed"],
+            "enum": ["proposed", "requested", "cancelled", "completed"],
```

Comment:
I think there is another file to update (the definitions/move.json)
They are basically duplicated and not sure why we do need both.

Reply:
I've deleted the duplicate and updated the one place that it was loaded

Reply:
🤞